src/pages/api/sysinfo.ts

```diff
@@ -12,5 +13,6 @@ export default function handler(req: NextApiRequest, res: NextApiResponse) {
     nodeEnv: __NODE_ENV__,
     analyticsUrl: process.env.MATOMO_URL,
     analyticsSiteId: process.env.MATOMO_SITE_ID,
+    keystoneUrl: __KEYSTONE_URL__,
```

Comment:
I noticed that I don't actually see any value here if the `NEXT_PUBLIC_KEYSTONE_URL` var is not defined until after building the image, and I think this needs to be treated like the Matomo vars:
```
keystoneUrl: process.env.NEXT_PUBLIC_KEYSTONE_URL
```

the pattern of injecting this value into the `next.config.js` file and using `delcare const` here is only for variables whose value is known at _build_ time (it's a webpack thing)